src/erlang_ls_parser.erl

```diff
@@ -60,8 +60,8 @@ parse_extra(IoDevice, Extra, StartLocation) ->
       end;
     {eof, _} ->
       {ok, Extra};
-    {error, Error} ->
-      {error, Error}
+    {error, ErrorInfo, EndLocation} ->
```

Comment:
This change is based on the error tuple that is returned from the call to `io:scan_erl_form/3`.